karyon3-examples/src/main/java/com/netflix/karyon/example/jetty/HelloWorldApp.java

```diff
@@ -27,44 +27,43 @@
     private static final Logger LOG = LoggerFactory.getLogger(HelloWorldApp.class);
     
     public static void main(String[] args) throws Exception {
-
-        Karyon.createInjector(
-            ArchaiusKaryonConfiguration.builder()
+        Karyon.builder()
+            .using(ArchaiusKaryonSuite.builder()
```

Comment:
so instead of "using", we can say
.withInitialConfig(CompositeConfig.builder().withConfigName("helloworld").withApplicationOverrides(...))
I haven't fully thought through how we can add this CompositeConfig to the layered hierarchy yet. But ultimately, I really think the relationship between Karyon and Configuration is composition instead of inheritance.